### DISCUSSION TITLE
Use mediamtx to convert udp stream to rtsp

### DISCUSSION THREAD

**Discussion by AntoineDubois12** [bluenviron/mediamtx#3703] [Questions]
### Question

I have an old camera that streams udp packets to my computer on port 1234, my computer and camera are in the same network (class c) and my computer firewall is off. I am trying to use Mediamtx to convert the udp stream to rtsp and publish the stream on path `/a1a7e359-8f71-428c-81aa-14765301d100`. When I go to my vlc and try to see the video from `rtsp://<mediamtx-ip>:<rtsp-port>/a1a7e359-8f71-428c-81aa-14765301d100` I can't see anything (I also don't get an error in the Mediamtx it seems that a connection is created but no stream is transferred. I use wireshark to catch the traffic of udp packages that come to my computer I attached below a pickup file of the udp stream and also the mediamtx.yml file that I use.
hope someone could help me with it
Mediamtx logs
![image](https://github.com/user-attachments/assets/ab6cd0e4-154c-4def-8ff9-e2e42733fd94)
my computer os is windows11 and I download MediaMTX v1.9.0

**Comment by aler9** [comment]
Hello, log message "[UDP source] ready: 1 track (H264)" means that you correctly configured the server and that the server is able to receive the UDP stream and recognize the H264 track.

If VLC is not showing anything, that probably depends on the content of the stream. In order to replicate the issue, you need to provide a network dump that contains the video stream. It can be generated in this way:
can be generated in this way:

1. Download wireshark (https://www.wireshark.org/)
2. Start capturing on the interface used for exchanging packets * If the server and the external hardware or software are both installed on your pc, the interface is probably "loopback", otherwise it's the one of your network card.
3. Start the server and replicate the issue
4. Stop capturing, save the result in .pcap format
5. Attach
